### PR TITLE
GEOMESA-2260 Update to latest Cloud Bigtable client

### PR DESCRIPTION
**Pull request by billyjacobson**
Updating Geomesa to use the latest Cloud Bigtable client and resolve bugs due to the old version

### CODE REVIEW

File: geomesa-bigtable/pom.xml [hunk 1]
```diff
@@ -49,11 +36,11 @@
...
             <modules>
                 <module>geomesa-bigtable-datastore</module>
-                <module>geomesa-bigtable-dist</module>
+                <!--<module>geomesa-bigtable-dist</module>-->
                 <module>geomesa-bigtable-gs-plugin</module>
                 <module>geomesa-bigtable-tools</module>
-                <module>geomesa-bigtable-spark</module>
-                <module>geomesa-bigtable-spark-runtime</module>
+                <!--<module>geomesa-bigtable-spark</module>-->
+                <!--<module>geomesa-bigtable-spark-runtime</module>-->
```
Comment (elahrvivaz) [inline geomesa-bigtable/pom.xml]: Thanks for the submission! Do these modules no longer work, or were they commented out accidentally?

Comment (billyjacobson) [reply geomesa-bigtable/pom.xml]: Oh, I was just having trouble rebuilding this locally so I commented them out. Will fix now.

**Comment by elahrvivaz** [comment]
Thanks again for the submission! I'm getting some errors building; for one, looks like we need to update the licenses... that can be done with `mvn license:format -f geomesa-bigtable/ -Pbigtable`. But also, I am getting:
```
[info] Compiling 1 Scala source and 3 Java sources to /opt/devel/src/geomesa/geomesa-bigtable/geomesa-bigtable-spark/target/classes...
[error] /opt/devel/src/geomesa/geomesa-bigtable/geomesa-bigtable-spark/src/main/java/org/locationtech/geomesa/bigtable/spark/BigtableInputFormatBase.java:28: error: package com.google.bigtable.v2 does not exist
[error] import com.google.bigtable.v2.RowSet;
[error]                              ^
[error] /opt/devel/src/geomesa/geomesa-bigtable/geomesa-bigtable-spark/src/main/java/org/locationtech/geomesa/bigtable/spark/BigtableInputFormatBase.java:210: error: cannot find symbol
[error]     RowSet rs = RowSet.parseFrom(rowsetbytes);
[error]     ^
[error]   symbol:   class RowSet
[error]   location: class BigtableInputFormatBase
[error] /opt/devel/src/geomesa/geomesa-bigtable/geomesa-bigtable-spark/src/main/java/org/locationtech/geomesa/bigtable/spark/BigtableInputFormatBase.java:210: error: cannot find symbol
[error]     RowSet rs = RowSet.parseFrom(rowsetbytes);
[error]                 ^
[error]   symbol:   variable RowSet
[error]   location: class BigtableInputFormatBase
[error] 3 errors
```

**Comment by elahrvivaz** [comment]
oh, looks like the license check is failing due to the indentation change in BigtableIndexAdapter.scala. If you revert that file it should pass again.

**Comment by billyjacobson** [comment]
@elahrvivaz fixed that :)

**Comment by billyjacobson** [comment]
@elahrvivaz Just wanted to follow up on this. Let me know if there's anything else to do.

**Comment by elahrvivaz** [comment]
@billyjacobson I think that there are still some compiler errors around `com.google.bigtable.v2.RowSet` in `BigtableInputFormatBase`. Our CI build doesn't actually build the bigtable modules, as we have them in a separate profile due to licensing of some of the transitive dependencies, so even though it passed I think the build is broken. To verify that it builds, you can run `mvn clean install -P bigtable -f geomesa-bigtable` - when I tried that I got the errors I posted before. Thanks,

**Comment by billyjacobson** [comment]
Sorry for the delay on all this, was busy with conference season! I finally got a chance to look at the error and fixed it + added the licenses @elahrvivaz

**Comment by elahrvivaz** [comment]
Thanks for the fix!

**Pull request merged**